c++/src/capnp/rpc.c++

```diff
@@ -4655,6 +4742,18 @@ private:
 
     // No more using `call` after this point, as it now belongs to the context.
 
+    if (call.getIsRealtime()) {
+      auto promiseAndPipeline = startCall(
+          call.getInterfaceId(), call.getMethodId(), kj::mv(capability), context->addRef(), hints);
+      tasks.add(promiseAndPipeline.promise
+          .attach(kj::mv(context))
+          .then([](){}, [](kj::Exception&& exception) {
+            KJ_LOG(ERROR, exception);
```

Comment:
Will this log errors thrown by the application?

Do we want it to? I guess on one hand errors shouldn't just be silently discarded, but on the other hand there's nowhere to report it since this is a realtime call...